java/maven/arch.xml

```diff
@@ -186,6 +193,20 @@
           to specify the default behavior of compile on save in Maven 
           projects.
       </api> 
+      <api category="devel" group="property" name="exec.vmArgs" type="export">
+          The plugin exports Java VM parameters to be used for application execution in <code>${exec.vmArgs}</code> 
+          property that can be used in action mappings or Maven pom.xml.
+      </api>
+      <api category="devel" group="property" name="exec.appArgs" type="export">
+          The plugin exports application parameters to be used for application execution in <code>${exec.appArgs}</code> 
+          property that can be used in action mappings or Maven pom.xml.
+      </api>
+      <api category="devel" group="lookup" name="ExplicitProcessParameters" type="export">
```

Comment:
In this case `devel` is fine (even I would prefer higher commitment ;-).

Reply:
Changed to `stable` in 9d7654ce75.